### PR TITLE
Avoid showing telemetry UI permissions twice and add tests

### PR DESCRIPTION
**Pull request by andresgutgon**
# What?
Fix this issue https://github.com/latitude-dev/latitude/issues/177

## 🐛 The bug: Double question
image

## Awesome test suite
image

### CODE REVIEW

File: packages/cli/src/lib/telemetry/index.ts [hunk 1]
```diff
@@ -41,33 +40,26 @@ class Telemetry {
...
 
   enable() {
-    this.track({ event: 'telemetryEnabled' })
+    this.trackWithoutCheck({ event: 'telemetryEnabled' })
     configStore.set('telemetry.enabled', true)
   }
 
   disable() {
-    this.track({ event: 'telemetryDisabled' })
+    this.trackWithoutCheck({ event: 'telemetryDisabled' })
```
Comment (andresgutgon) [inline packages/cli/src/lib/telemetry/index.ts:55]: This was the bug. We were calling `track` which has `setup` inside so inside a setup we ended calling again `setup` and there for the question appeared twice

File: packages/cli/src/lib/telemetry/index.ts [hunk 2]
```diff
@@ -41,33 +40,26 @@ class Telemetry {
...
   private async setup() {
-    // Only do this one time one running the CLI
     if (this.initialized) return this.enabled
-
-    if (this.enabled !== undefined) {
-      this.initialized = true
-      return this.enabled
-    }
+    if (this.enabled === true) return true
+    if (this.enabled === false) return false
```
Comment (csansoon) [inline packages/cli/src/lib/telemetry/index.ts:62]: `if (this.enabled !== undefined) return this.enabled`

File: packages/cli/src/lib/telemetry/index.ts (hunk 1 above)
Comment (csansoon) [reply packages/cli/src/lib/telemetry/index.ts:55]: Also now that you're with it, there's another bug where sometimes it doesn't add a new line after the last selectable option, so the next message appears right after "No, maybe later\<next message>":

image

**Review by csansoon — DISMISSED** [review]

File: packages/cli/src/lib/telemetry/index.ts (hunk 1 above)
Comment (andresgutgon) [reply packages/cli/src/lib/telemetry/index.ts:55]: This is also fixed with this

**Pull request merged**